plugins/kscript

```diff
@@ -0,0 +1 @@
+repository = https://github.com/edgelevel/asdf-kscript
```

Comment:
This should be the URL to the Git repository itself, which for Github project would be `https://github.com/edgelevel/asdf-kscript.git`.

Reply:
Done